lib/core/error.js

```diff
@@ -207,7 +207,9 @@ function isNodeShuttingDownError(err) {
  * @param {Server} server
  */
 function isSDAMUnrecoverableError(error, server) {
-  if (error instanceof MongoParseError) {
+  // NOTE: null check is here for a strictly pre-CMAP world, a timeout or
```

Comment:
ticket for the 4.0 epic?

Reply:
https://jira.mongodb.org/browse/NODE-2349